lib/Indexer/Tests/Adapter/IndexBuilderTestCase.php

```diff
@@ -38,6 +39,24 @@ public function testIndexClass(string $source, string $name, Closure $assertions
      */
     public function provideIndexesClassLike(): Generator
     {
+        yield 'attribute' => [
+            <<<DOC
+                    // File: project/attribute.php
+                    <?php
+
+                    #[Attribute]
+                    class IamAnAttribute {}
+                DOC,
+            'IamAnAttribute',
+            function (ClassRecord $record): void {
+                self::assertInstanceOf(ClassRecord::class, $record);
+                self::assertEquals($this->workspace()->path('project/attribute.php'), $record->filePath());
+                self::assertEquals('IamAnAttribute', $record->fqn());
+                self::assertEquals(15, $record->start()->toInt());
+                self::assertEquals(HasFlags::ATTRIBUTE, $record->flags());
```

Comment:
`self::assertTrue($record->flags() & HasFlags::ATTRIBUTE)` would be more fitting for the usage.